packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -180,8 +190,16 @@ export class MenusContributionPointHandler {
         const command: Command = { id };
         this.commands.registerCommand(command, handler);
 
-        const { group, when } = action;
-        const item: Mutable<TabBarToolbarItem> = { id, command: id, group: group || '', when };
+        const { when } = action;
+        // handle group and priority
```

Comment:
I'd update the documentation to something similar to:

```ts
// Handle the group and priority of a context menu item.
// The expected format consists of the following: '{string}@{number}' where {string} is the group name, and {number} is the priority.
// * If the provided group is empty or only contains whitespace character(s), the group will be set to 'navigation'
// * If the priority is no a number it will be set to 0
// Example:
// * 'navigation@1' => ['navigation', 1]
// * 'test@2' => ['test', 2]
// * '@1' => ['navigation', 1]
// * 'test@test' => ['test', 0]
```